src/full/Agda/Syntax/Translation/ConcreteToAbstract.hs

```diff
@@ -1918,7 +1918,7 @@ instance ToAbstract NiceDeclaration A.Declaration where
       zipWithM_ (rebindName p OtherDefName) xs ys
       return [ A.UnquoteDef [ mkDefInfo x fx PublicAccess a r | (fx, x) <- zip fxs xs ] ys e ]
 
-    NicePatternSyn r a n as p -> do
+    NicePatternB r ps -> fmap join $ forM ps $ \ (NicePatternSyn r a n as p) -> do
```

Comment:
So should I read this as “just do what we always did, but iterate over the (nice) declarations inside”?

Reply:
Yep.